test/truffle/docker/README.md

```diff
@@ -13,8 +13,11 @@ You need to put the built GraalVM binary tarball into the same directory as the
 Dockerfile, and update the Dockerfile for the version.
 
 ```
-$ docker build -t truffleruby-test-ubuntu .
+$ docker build -t truffleruby-test-ubuntu . --build-arg GRAALVM_VERSION=n.n
```

Comment:
Should `--build-arg GRAALVM_VERSION=n.n` before the `.`?
If it works as is then it likely does not matter.

Reply:
It works like this.